ReadMe.rst

```diff
@@ -18,44 +18,33 @@ Branch Status - release/202405
   In Development
 
 :Entered Development:
-  Todo
+  2023/11/24
 
 :Anticipated Stabilization:
-  Todo
+  Nov 2024
 
 Branch Changes - release/202405
 ===============================
 
+202405 is a larger deviation than previous releases. As part of upstreaming changes to EDK2, the commits were reviewed, squashed, and some were dropped.
+Due to these changes, there maybe more work minor work required to bring an existing platforms up to 202405 compatibality. 
+
 Breaking Changes-dev
 --------------------
+- ArmPlatformPkg\Drivers\Apei\HestDxe\HestDxe.inf has been dropped.
+- ArmPlatformPkg\Drivers\HestMmErrorSources\HestErrorSourceDxe.inf has been dropped.
+- ArmPlatformPkg\Include\Protocol\HestErrorSourceInfo.h has been dropped.
+- ArmPlatformPkg\Include\Protocol\HestTable.h has been dropped.
+- ArmVirtPkg\Include\Library\FdtSerialPortAddressLib.h has been dropped.
+- ArmVirtPkg\Library\FdtSerialPortAddressLib\FdtSerialPortAddressLib.inf has been dropped.
+- ArmVirtPkg\Library\PlatformBootManagerLib\PlatformBootManagerLib.inf has been dropped. 

```

Comment:
From my side, the extended functionality from `ArmGicLib` that used to be in 2311 is moved its own library `MsCorePkg/Library/MuArmGicExLib` in mu_plus.

Reply:
Thanks, it should be captured now.